test_setup.alerts_logger.yml

```diff
@@ -0,0 +1,36 @@
+---
+# ========================================
+#  QE configuration for alerts logger
+# ========================================
+#
+# This playbook serves as setup of alerts logger on tendrl server machine.
+# It periodically logs alerts from tendrl api based on tendrl user.
+
+- hosts: usm_server
+  remote_user: root
+  vars:
+    tendrl_user: "admin"
+    tendrl_password: "adminuser"
+
+  tasks:
+    - name: Create alerts_logger python script based on a template
+      template:
+        dest: /usr/bin/alerts_logger_{{ tendrl_user }}.py
```

Comment:
I would rather put it into `/usr/loca/bin` (which is better suited for executables which are not installed via rpm).